internal/service/cloudwatch/composite_alarm_test.go

```diff
@@ -626,3 +657,26 @@ resource "aws_cloudwatch_composite_alarm" "test" {
 }
 `, rName))
 }
+
+func testAccCompositeAlarmConfig_actionSuppressor(rName string) string {
+	return acctest.ConfigCompose(testAccCompositeAlarmConfig_base(rName), fmt.Sprintf(`
+resource "aws_sns_topic" "test" {
+  count = 3
+  name  = "%[1]s-${count.index}"
+}
+
+resource "aws_cloudwatch_composite_alarm" "test" {
+  alarm_actions             		  = [aws_sns_topic.test[0].arn]
+  alarm_name                		  = %[1]q
+  alarm_rule                		  = "ALARM(${aws_cloudwatch_metric_alarm.test[0].alarm_name})"
+  insufficient_data_actions 		  = [aws_sns_topic.test[1].arn]
+  ok_actions                		  = [aws_sns_topic.test[2].arn]
+
+  actions_suppressor {
+	alarm  		  		= "${aws_cloudwatch_metric_alarm.test[0].alarm_name}"
```

Comment:
Can't this just be `aws_cloudwatch_metric_alarm.test[0].alarm_name` ?

Reply:
Yes it can, changed it.